rpm/Dockerfile

```diff
@@ -7,5 +7,6 @@ RUN rpmdev-setuptree
 
 USER root
 ADD entry.sh /root/
+COPY ./ /root/rpmbuild/SPECS
```

Comment:
The slowdown is almost nonexistant if you have no local files. There's no way to work around it with some sort of magic docker mount flag.. or else we pollute the host system after the build.